test/services/EmploymentServiceSpec.scala

```diff
@@ -92,11 +93,45 @@ class EmploymentServiceSpec extends TestUtils {
           .expects(nino, taxYear, employmentId, *)
           .returning(Future.successful(Left(desError)))
 
-        val result = createEmploymentService.deleteEmployment(nino, taxYear, employmentId)
+        val result = employmentService.deleteEmployment(nino, taxYear, employmentId)
 
         await(result) mustBe Left(desError)
       }
     }
   }
 
+  "deleteEmploymentFinancialData" should {
+
+    "return Right" when {
+
+      "deleteEmploymentFinancialData connector call succeeds" in {
+
+        (mockDeleteEmploymentFinancialDataConnector.deleteEmploymentFinancialData(_: String, _: Int, _: String)(_: HeaderCarrier))
+          .expects(nino, taxYear, employmentId, *)
+          .returning(Future.successful(Right(())))
+
+        val result = employmentService.deleteEmploymentFinancialData(nino, taxYear, employmentId)
+
+        await(result) mustBe Right(())
+      }
+    }
+
+    "return Left containing DES Error" when {
+
+      "the deleteEmploymentFinancialData connector fails" in {
+
+        val desError = DesErrorModel(500, DesErrorBodyModel("DES_CODE", "DES_REASON"))
```

Comment:
can be changed to INTERNAL_SERVER_ERROR to get rid of magic number warning

Reply:
there is another line length warning in here as well.

Reply:
Sure